Author: Dave Jones <dave.jones@canonical.com>
Forwarded: yes
Description: Revert commit c6b9fbf7566f84a807a5c116288648085fa529df
 This commit adds SDHCI DMA support which works happily with the SD card
 interface on the Pi 4, but breaks eMMC support on the CM4

--- a/configs/rpi_4_32b_defconfig
+++ b/configs/rpi_4_32b_defconfig
@@ -26,7 +26,6 @@
 CONFIG_DM_KEYBOARD=y
 CONFIG_DM_MMC=y
 CONFIG_MMC_SDHCI=y
-CONFIG_MMC_SDHCI_SDMA=y
 CONFIG_MMC_SDHCI_BCM2835=y
 CONFIG_DM_ETH=y
 CONFIG_BCMGENET=y
--- a/configs/rpi_4_defconfig
+++ b/configs/rpi_4_defconfig
@@ -26,7 +26,6 @@
 CONFIG_DM_KEYBOARD=y
 CONFIG_DM_MMC=y
 CONFIG_MMC_SDHCI=y
-CONFIG_MMC_SDHCI_SDMA=y
 CONFIG_MMC_SDHCI_BCM2835=y
 CONFIG_DM_ETH=y
 CONFIG_BCMGENET=y
